index.Rmd

```diff
@@ -389,6 +393,8 @@ c <- rast(ncol = 10, nrow = 10)
 class(c)
 ```
 
+Here we used the function `rast` from the Terra package to create an object of class `SpatRaster`. `SpatRaster` is a class for rasters (or matrices) that comprise the necessary elements for spatial data. With Terra and `SpatRaster`, complex spatial data will be processed more efficiently by using specifically tailored functions for `SpatRaster`. This will be studied further in [another tutorial](../IntroToRaster/index.html).
```

Comment:
The code above uses the Terra package and the SpatRaster class. I added a short description for context even though this will be covered in later tutorials. However, this paragraph is not necessary if we want to make the tutorial shorter.